Validation/CaloTowers/test/run_onRelVal_data_cfg.py

```diff
@@ -0,0 +1,152 @@
+import os
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("hcalval")
+process.load("Configuration.StandardSequences.Reconstruction_cff")
+process.load("Configuration.StandardSequences.Geometry_cff")
```

Comment:
We will be preparing another pull request for further HCAL relval 
changes in the near future.  We will address this in this pull request.

Joe

On 4/8/2015 1:57 PM, David Lange wrote:
>
> In Validation/CaloTowers/test/run_onRelVal_data_cfg.py 
> <https://github.com/cms-sw/cmssw/pull/8657#discussion_r28001690>:
>
> > @@ -0,0 +1,152 @@
> > +import os
> > +import FWCore.ParameterSet.Config as cms
> > +
> > +process = cms.Process("hcalval")
> > +process.load("Configuration.StandardSequences.Reconstruction_cff")
> > +process.load("Configuration.StandardSequences.Geometry_cff")
>
> This is the wrong geometry file to load. Please replace it with 
> GeometryRecoDB_cff instead.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/cms-sw/cmssw/pull/8657/files#r28001690>.
>